### ISSUE TITLE
Grid summary cells should be navigable by the GridNavigation

### ISSUE THREAD

**Issue by ExtAnimal** [bryntum/support#9414]
[Forum post](https://forum.bryntum.com/viewtopic.php?t=29468)

Hi,

We are currently exploring the Grid.feature.Summary feature in our Grid.

We've noticed that the summary row can't be accessed when using a screen reader. Is this something that you could add support for please?

Thanks!
